src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -1380,26 +1369,18 @@ void SystemDictionary::define_instance_class(InstanceKlass* k, Handle class_load
   ClassLoaderData* loader_data = k->class_loader_data();
   assert(loader_data->class_loader() == class_loader(), "they must be the same");
 
-  // Bootstrap and other parallel classloaders don't acquire a lock,
-  // they use placeholder token.
-  // If a parallelCapable class loader calls define_instance_class instead of
+  // Class loaders do not acquire a lock, they use placeholder token.
+  // If a class loader calls define_instance_class instead of
   // find_or_define_instance_class to get here, we have a timing
   // hole with systemDictionary updates and check_constraints
```

Comment:
AFAICS the only call to `define_instance_class` is now through `find_or_define_instance_class` so this comment no longer makes much sense to me.

Reply:
It seems that the comment was describing a bug someone once had.  I can remove it.